server/views/layout.njk

```diff
@@ -56,8 +56,6 @@
   <![endif]-->
 
   <script src="/assets/js/jquery.min.js"></script> 
-  <script src="https://code.jquery.com/ui/1.13.1/jquery-ui.min.js" integrity="sha256-eTyxS0rkjpLEo16uXTS0uVCS4815lc40K2iVpWDvdSY=" crossorigin="anonymous"></script>
```

Comment:
Also removed jquery UI from the page template whilst I was here - it appears to have had no ill effects 😁